test/cluster/conftest.py

```diff
@@ -265,9 +270,11 @@ async def manager(request: pytest.FixtureRequest,
     await manager_client.stop()  # Stop client session and close driver after each test
     if cluster_status["server_broken"]:
         pytest.fail(
-            f"test case {test_case_name} leave unfinished tasks on Scylla server. Server marked as broken,"
+            f"test case {test_case_name} left unfinished tasks on Scylla server. Server marked as broken,"
             f" server_broken_reason: {cluster_status["message"]}"
         )
+    if found_errors:
+        pytest.fail("\n" + found_errors)
```

Comment:
Here, I think, it will be enough to put something like this:
```
Found 6 critical error, 3 errors and core dump. For more information check the file: {file_path}
```
The file with all that information you can attach to the `failed_test` directory in the if-clause in the line 250. Of course, some code reorganization will be needed.